snorkel/labeling/model/label_model.py

```diff
@@ -400,9 +409,14 @@
                [0., 1.],
                [0., 1.]])
         """
-        L_shift = L + 1  # convert to {0, 1, ..., k}
-        self._set_constants(L_shift)
-        L_aug = self._get_augmented_label_matrix(L_shift)
+        if not is_augmented:
+            # This is the usual mode
+            L_shift = L + 1  # convert to {0, 1, ..., k}
+            self._set_constants(L_shift)  # TODO - Why do we need this here ?
```

Comment:
self._get_augmented_label_matrix uses at least self.cardinality, which is set in this method. Remove the TODO?